setup.py

```diff
@@ -1,22 +1,36 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-import setuptools
+from setuptools import setup, find_packages
+from os.path import join as opj, dirname
+import re
 
-#
-# Grab some info from supporting files
-#
-with open('README.md', 'r') as fh:
-    long_description = fh.read()
+version_file = 'htcrystalball/_version.py'
+readme = opj(dirname(__file__), 'README.md')
+mo = re.search(
+    r"^__version__ = ['\"]([^'\"]*)['\"]",
+    open(version_file, "rt").read(),
+    re.M
+)
+
+if mo:
+    version = mo.group(1)
+else:
+    raise RuntimeError('Unable to find version string in %s.' % version_file)
+
+try:
+    import pypandoc
```

Comment:
Is there a motivation for doing this? According to the [setup.py docs](https://packaging.python.org/guides/making-a-pypi-friendly-readme/), Markdown is supported natively.

I'm curious what the advantage is to converting to .rst first. Otherwise, I don't see the advantage of introducing a large dependency.

Reply:
Its actually not a dependency (it is used only if the package is already installed). The reason behind this is that some online resources (like PyPi) support rich text instead of markdown. If you think this is not necessary I will remove it.